spec/support/active_job.rb

```diff
@@ -1,3 +1,8 @@
 RSpec.configure do |config|
   config.include(ActiveJob::TestHelper)
+
+  config.before(:each) do
+    ActiveJob::Base.queue_adapter.enqueued_jobs = []
+    ActiveJob::Base.queue_adapter.performed_jobs = []
+  end
```

Comment:
The `ActiveJob::TestHelper` has a [`before_setup`][1] method that clears both of these. If you're thinking but that's Minitest and not RSpec then don't worry - RSpec Rails calls it [here][2]. :smile:

[1]: https://github.com/rails/rails/blob/master/activejob/lib/active_job/test_helper.rb#L22-L23
[2]: https://github.com/rspec/rspec-rails/blob/master/lib/rspec/rails/adapters.rb#L126